dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -57,11 +57,17 @@ ERROR_CONFIG_EMPTY_LIBRARY=SESN0309E: The {0} session cache library is empty. {1
 ERROR_CONFIG_EMPTY_LIBRARY.explanation=The sessionCache feature could not find a JCache provider to load.
 ERROR_CONFIG_EMPTY_LIBRARY.useraction=Verify that the library contains a JCache provider and all paths are specified correctly.
 
+# !!!! MOVED TO IO.OPENLIBERTY.JCACHE.INTERNAL !!!!
 # do not translate: httpSessionCache, uri
-SESN0310_GEN_INFINISPAN_CONFIG=SESN0310I: Generated Infinispan configuration for HTTP session persistence: {0}. To override, configure the uri attribute of the httpSessionCache configuration element.
-SESN0310_GEN_INFINISPAN_CONFIG.explanation=The Infinispan caches that are used for HTTP session persistence require additional configuration to persist sessions across servers. Liberty generates this configuration when it is not provided by the user.
-SESN0310_GEN_INFINISPAN_CONFIG.useraction=No action is required if you are satisfied with the generated configuration. Otherwise, the generated configuration can be used as a starting point.
+#SESN0310_GEN_INFINISPAN_CONFIG=SESN0310I: Generated Infinispan configuration for HTTP session persistence: {0}. To override, configure the uri attribute of the httpSessionCache configuration element.
+#SESN0310_GEN_INFINISPAN_CONFIG.explanation=The Infinispan caches that are used for HTTP session persistence require additional configuration to persist sessions across servers. Liberty generates this configuration when it is not provided by the user.
+#SESN0310_GEN_INFINISPAN_CONFIG.useraction=No action is required if you are satisfied with the generated configuration. Otherwise, the generated configuration can be used as a starting point.
+
+CONFIG_ATTRIBUTE_IGNORED=SESN0311W: The {0} configuration attribute was specified, but is ignored because the jCacheManagerRef attribute was specified.
+CONFIG_ATTRIBUTE_IGNORED.explanation=The specified configuration is ignored as it is provided by the specified jCacheManagerRef.
```

Comment:
@jvanhill 
Since no action is required, please change the message to an information message ( SESN0311W --> SESN0311I ).